charts/posit-chronicle/values.yaml

```diff
@@ -0,0 +1,85 @@
+image:
+  repository: "935931255537.dkr.ecr.us-east-2.amazonaws.com/chronicle"
+  tag: "latest"
+  imagePullPolicy: "Always"
+
+serviceaccount:
+  enabled: false
```

Comment:
Nit: should we use `create` here instead to be more consistent with the other charts?

Reply:
sure we can do that, consistency should be the main goal and on reflection i do think `serviceaccount.create` is more clear than `serviceaccount.enable`.

`serviceaccount.createAndUse` might be most accurate, but that's a little much and not in line with our other charts.